stl/inc/chrono

```diff
@@ -6202,6 +6163,128 @@ namespace chrono {
             }
         }
 
+        template <class _Ty>
+        void _Validate_specifiers(const _Chrono_spec<_CharT>& _Spec, const _Ty& _Val) {
+            // clang-format off
+            if constexpr (
+                _Is_specialization_v<_Ty, duration>
+                || is_same_v<_Ty, sys_info> || _Is_specialization_v<_Ty, time_point>
+                || _Is_specialization_v<_Ty, _Local_time_format_t>) {
+                return;
+            }

```

Comment:
Clang format messes this up, but it can be formatted a little more inline (a few clauses got removed over time).

My understanding is that these types are _always_ valid by design, so they just work.